frontend/src/metabase/visualizations/components/LeafletChoropleth.jsx

```diff
@@ -49,7 +49,7 @@ const LeafletChoropleth = ({
         fillColor: getColor(feature),
         weight: 1,
         opacity: 1,
-        color: "white",
+        color: getColor("white"),
```

Comment:
I don't think this is how `getColor` is meant to be used. You can see that it accepts `feature` not a color.

https://github.com/metabase/metabase/blob/0768495c06cb1e8a35ab704d26b6f3f58ac48f13/frontend/src/metabase/visualizations/components/ChoroplethMap.jsx#L354-L357


I think we should revert this change.